src/lib/Content/Form/Provider/GroupedContentFormFieldsProvider.php

```diff
@@ -25,16 +25,13 @@ public function __construct(FieldsGroupsList $fieldsGroupsList)
 
     public function getGroupedFields(array $fieldsDataForm): array
     {
-        $fieldsGroups = $this->fieldsGroupsList->getGroups();
         $groupedFields = [];
 
         foreach ($fieldsDataForm as $fieldForm) {
             /** @var \Ibexa\Contracts\ContentForms\Data\Content\FieldData $fieldData */
             $fieldData = $fieldForm->getViewData();
             $fieldGroupIdentifier = $this->fieldsGroupsList->getFieldGroup($fieldData->fieldDefinition);
-            $fieldGroupName = $fieldsGroups[$fieldGroupIdentifier] ?? $this->fieldsGroupsList->getDefaultGroup();
```

Comment:
Even tho it seems like a bug, not sure we can fix it that way due to bc policy. Otoh, this seems so internal and if only used by that one place (is it tho?) it should be acceptable. 
@ibexa/php-dev